tests/aws/services/sqs/test_sqs_move_task.py

```diff
@@ -0,0 +1,452 @@
+import json
+import time
+import uuid
+
+import pytest
+from botocore.exceptions import ClientError
+
+from localstack.constants import TEST_AWS_REGION_NAME
+from localstack.services.sqs.utils import decode_move_task_handle, encode_move_task_handle
+from localstack.testing.pytest import markers
+from localstack.utils.aws import arns
+from localstack.utils.sync import poll_condition, retry
+
+QueueUrl = str
+
+
+def sqs_collect_messages(
+    sqs_client, queue_url: str, expected: int, timeout: int, delete: bool = True
+):
+    collected = []
+
+    def _receive():
+        response = sqs_client.receive_message(
+            QueueUrl=queue_url,
+            WaitTimeSeconds=min(max(1, int(timeout)), 5),
```

Comment:
I don't quite understand WaitTimeSeconds.
a) nit: it's an int regarding to the type hint, do we really need the cast?
b) There are multiple invocations with timeout=60 in the tests if I see this correctly. However, `min(max(1, timeout),5) will always return a number between 1 and 5, so why do we repeatedly invoke with 60?

Reply:
a) good point!
b) yea it's maybe a bit confusing what i did here. just tried to find a trade off of not polling too often, and not waiting too long on each poll